backend/core/serializers.py

```
@@ -9,19 +9,31 @@ class Meta:
+                    'definition', 
+                    'sentence_jp', 
+                    'sentence_en', 
+                    'word_img', 
+                    'word_audio', 
+                    'sentence_audio', 
+                ]
+
+
+class CardSerializer(serializers.ModelSerializer):
     class Meta:
         model = Card
```

Ruff

`Card` may be undefined, or defined from star imports

(F405)